ignite/engine/events.py

```diff
@@ -71,14 +71,17 @@ def __call__(
             CallableEventWithFilter: A new event having the same value but a different filter function
         """
 
-        if (event_filter is not None) and (
-            once is not None or every is not None or before is not None or after is not None
-        ):
-            raise ValueError("Only One of the Input Arguments should be Specified, except Before, after and Every")
-        if not (event_filter is not None) and not (
-            once is not None or every is not None or before is not None or after is not None
+        if (
+            sum(
+                (
+                    event_filter is not None,
+                    once is not None,
+                    (every is not None or before is not None or after is not None),
+                )
+            )
+            != 1
         ):
-            raise ValueError("All of the Arguments cannot be undefined")
+            raise ValueError("Only one of the Input Arguments should be Sepcified, except before, after and every")
```

Comment:
@theory-in-progress please rewrite the error message and fix typos and capitalize only the word of the sentence.